login-workflow/src/screens/AuthNavigationContainer.tsx

```diff
@@ -72,9 +72,14 @@ export const AuthNavigationContainer: React.FC<NavigationContainerComponentProps
     }, []);
 
     const { routes, routesArray: publicRoutes } = prefixRoutes({ ...defaultRoutes, ...routeConfig });
+    const extraRoutesPathnames = extraRoutes.map((route) => route.props.path);
 
     // Show the splash loading screen until we have loaded the initial authentication state
-    if (securityState.isLoading) {
+    if (
+        securityState.isLoading &&
+        !publicRoutes.includes(window.location.pathname) &&
```

Comment:
Looks good, I would just take out this line. We will want to show the loading spinner when you are accessing one of the login-workflow screens or a private route. Just not for public (extra) routes.
```suggestion
```